From: Andrew Morton <akpm@linux-foundation.org>
Subject: sparc-mm-export-symbols-for-lazy_mmu_mode-kunit-tests-fix
Date: Wed Jan 21 09:06:56 AM PST 2026

remove mm/tests/lazy_mmu_mode_kunit.c comment, per Kevin

Cc: Andreas Larsson <andreas@gaisler.com>
Cc: Kevin Brodsky <kevin.brodsky@arm.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/tests/lazy_mmu_mode_kunit.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/tests/lazy_mmu_mode_kunit.c~sparc-mm-export-symbols-for-lazy_mmu_mode-kunit-tests-fix
+++ a/mm/tests/lazy_mmu_mode_kunit.c
@@ -2,7 +2,6 @@
 #include <kunit/test.h>
 #include <linux/pgtable.h>
 
-/* For some symbols referenced by arch_{enter,leave}_lazy_mmu_mode on powerpc */
 MODULE_IMPORT_NS("EXPORTED_FOR_KUNIT_TESTING");
 
 static void expect_not_active(struct kunit *test)
_
